homeassistant/components/aladdin_connect/cover.py

```diff
@@ -75,11 +75,13 @@ async def async_will_remove_from_hass(self) -> None:
 
     async def async_close_cover(self, **kwargs: Any) -> None:
         """Issue close command to cover."""
-        await self._acc.close_door(self._device_id, self._number)
+        if not await self._acc.close_door(self._device_id, self._number):
```

Comment:
I assume these calls in themselves does never raise but always return something?

Reply:
They do not raise  - true if it worked, false if it doesn't.